main/app/services/chromadb_service.py

```diff
@@ -0,0 +1,87 @@
+import chromadb
+
+from typing import List, Any, Optional
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_chroma import Chroma
+from app.core.config import settings
+
+
+class ChromaVectorStore:
+    def __init__(
+        self,
+        collection_name: str,
+        embedding_function: Optional[Embeddings] = None,
+    ):
+        """Initialize Chroma vector store."""
+        self._chroma_client = chromadb.HttpClient(
+            host=settings.chroma_db_host,
+            port=settings.chroma_db_port,
+        )
+
+        self._store = Chroma(
+            client=self._chroma_client,
+            collection_name=collection_name,
+            embedding_function=embedding_function,
+        )
+
+    def add_documents(self, documents: List[Document]) -> None:
+        self._store.add_documents(documents)
```

Comment:
Unless I'm missing something we need the batching logic I added to improve document ingestion stability:

https://github.com/akvo/akvo-rag/blob/main/backend/app/services/vector_store/chroma.py#L28

Reply:
Yes, I will do it, can I do it in the next PR?
A lot of update already done in Feature/5 PR.